packages/terra-notification-dialog/src/_NotificationIcon.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './NotificationIcon.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const variants = {
```

Comment:
I don't think this object is really proving any value. I think it would be fine to just use the strings.

Reply:
removed: https://github.com/cerner/terra-framework/pull/1105/commits/022584400b4aaafcf4ad53ff1ed2e2b2eaec08a5#diff-5ea24c4951096e61fc7f083913af0c3fR16